homeassistant/components/here_travel_time/sensor.py

```diff
@@ -240,6 +240,11 @@ def __init__(
         self._attr_native_unit_of_measurement = TIME_MINUTES
         self._attr_name = name
 
+    async def async_added_to_hass(self) -> None:
+        """Wait for start so origin and destination entities can be resolved."""
+        await super().async_added_to_hass()
+        async_at_start(self.hass, self.async_update)
```

Comment:
async_at_start will pass a `HomeAsssistant` object to the callback which we don't need:
```suggestion
        async_at_start(self.hass, lambda hass: self.async_update())
```